### PR TITLE
mobile hamburger menu, compact hotkey

### LINKED ISSUE #200: Immersion UI: mobile hamburger menu
Almost forgot this one but it'll be handy for viewing previews on your mobile device.

Screen Shot 2022-09-15 at 3 56 53 PM

### PR DESCRIPTION
**Pull request by psugihara**
fixes #200

### CODE REVIEW

**Comment by psugihara** [comment]
hm, needs fixing

Screen Shot 2022-09-16 at 9 00 30 AM

File: packages/cli/src/components/hooks/usePreviewHotkeys.tsx [hunk 1]
```diff
@@ -69,13 +76,5 @@ export default function usePreviewHotkeys({ setViewMode }: Options) {
     [handleKey]
   );
 
-  const textareaRef = useCallback(
```
Comment (psugihara) [inline packages/cli/src/components/hooks/usePreviewHotkeys.tsx:72]: didn't realize but we don't need this and it sends double keys... I think this changed when we made it non-editable

File: packages/cli/src/components/IndexPane/IndexPane.tsx [hunk 2]
```diff
@@ -31,6 +31,8 @@ const IndexPane: React.FC<IndexPaneProps> = ({ previews, previewText }) => {
         right();
       } else if (compact && e.key === " " && treeRoutes) {
         setCollapse(cursor, !treeRoutes[cursor].collapsed);
+      } else if (e.key === "`") {
+        setCompact((current) => !current);
```
[the remark this answers is not in the document]
Comment (psugihara) [reply packages/cli/src/components/IndexPane/IndexPane.tsx:35]: nvm :)

**Comment by psugihara** [comment]
i *think* this is good to merge now

**Review by alexfarrill — APPROVED** [review]
Looks awesome! 🚀

**Pull request merged**